Makefile

```diff
@@ -286,7 +286,7 @@ GIT_VERSION: force
 
 include Makefile.docker
 
-CRD_OPTIONS ?= "crd:trivialVersions=true"
+CRD_OPTIONS ?= "crd:crdVersions=v1beta1"
```

Comment:
is this line relevant?

Reply:
Yes it is relevant because bumping to the latest controller-tools will cause v1 CRDs to be generated. Since we are still on v1beta1 CRDs, we need this line. The PR to migrate to v1 CRDs is https://github.com/cilium/cilium/pull/11477. (It is also the PR that I was working on where I likely messed up the SHAs due to force-pushing.)